CONTRIBUTING.md

```diff
@@ -6,10 +6,14 @@ To contribute to the guide, fork this repository and make the necessary ammendme
 
 ## Self-hosting
 
-To host the guide on GitHub Pages, the quickest way is by enabling it in the settings of GitHub. Set GitHub Pages to render from the `master` branch (refer [here](https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site)). Once this is done, GitHub Pages will take a while to upload your changes and it will soon be available at `https://<username>.github.io/git-guide`.
+To host the guide on GitHub Pages, the quickest way is by enabling it in the settings of GitHub. Set GitHub Pages to render from the `gh-pages` branch (refer [here](https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site)). Once this is done, GitHub Pages will take a while to upload your changes and it will soon be available at `https://<username>.github.io/git-guide`.
```

Comment:
`gh-pages` is the branch that hosts the website, but how do we get github to generate the static website like u did from publish? (I am assuming the `publish` branch has some workflow to build and push the static webpage to `gh-pages`)  

Do we need to add this info in?

Reply:
We don't need to include this for the CONTRIBUTING.md. The way the page deploys isn't super important but if you want to include it, I have a blog post detailing how the entire things works already: https://woojiahao.github.io/blog/posts/2020-04-14-Gatsby.js%20+%20GitHub%20Actions%20+%20GitHub%20Pages%20=%20Match%20Made%20in%20Heaven.

Reply:
Though, I'd say that this probably goes better in the `site/`'s README more than CONTRIBUTING.md

Reply:
Yea I was thinking about that as well because I don't think without building the site, it will show up on `gh-pages`.  
I might be wrong on this, let me check